migrations/helpers/update_filtered_positives_helper.py

```diff
@@ -243,17 +243,18 @@ def update_dart_fields(config: ModuleType, samples: List[Sample]) -> bool:
                 )
                 if plate_state == DART_STATE_PENDING:
                     for sample in samples_in_plate:
-                        well_index = get_dart_well_index(sample.get(FIELD_COORDINATE, None))
-                        if well_index is not None:
-                            well_props = map_mongo_doc_to_dart_well_props(sample)
-                            set_dart_well_properties(
-                                cursor, plate_barcode, well_props, well_index  # type:ignore
-                            )
-                        else:
-                            raise ValueError(
-                                "Unable to determine DART well index for sample "
-                                f"{sample[FIELD_ROOT_SAMPLE_ID]} in plate {plate_barcode}"
-                            )
+                        if sample[FIELD_RESULT] == POSITIVE_RESULT_VALUE:
```

Comment:
Remind me why we don't just check for filtered positive here?

Reply:
It stems from updating the filtered positive rules: if an unpicked sample is changed from fit to pick to not fit to pick, we wouldn't end up updating that sample in DART, as it wouldn't be filtered positive

Reply:
Oh yes, that rings a bell, thank you Chris.